plugins/parquet/parquetembed.cpp

```diff
@@ -1384,15 +1384,10 @@ void bindBoolParam(bool value, const RtlFieldInfo *field, std::shared_ptr<Parque
  */
 void bindDataParam(unsigned len, const char *value, const RtlFieldInfo *field, std::shared_ptr<ParquetHelper> r_parquet)
 {
-    size32_t utf8chars;
-    rtlDataAttr data;
-    rtlStrToUtf8X(utf8chars, data.refstr(), len, value);
-    size32_t utf8len = rtlUtf8Size(utf8chars, data.getstr());
-
     rapidjson::Value key;
     key.SetString(field->name, jsonAlloc);
     rapidjson::Value val;
-    val.SetString(data.getstr(), utf8len, jsonAlloc);
+    val.SetString(value, len, jsonAlloc);
```

Comment:
Still need

size32_t utf8size = rtlUtf8Size(len, value);	

Better to name it size rather than len to avoid confusion over whether it is a number of codepoints or a size.